docs/getstarted/keybindings.md

```diff
@@ -220,7 +220,7 @@ Key|Command|Command id
 `kb(workbench.action.debug.stepOut)`|Step Out|`workbench.action.debug.stepOut`
 `kb(workbench.action.debug.stepOver)`|Step Over|`workbench.action.debug.stepOver`
 `kb(workbench.action.debug.stop)`|Stop|`workbench.action.debug.stop`
-`kb(editor.action.showHover)`|Show Hover|`editor.action.showHover`

```

Comment:
I take this comment back. The section is on debugging commands so your change is correct. I'll add general showHover to the Rich Language Editing list.